test/e2e-tests.sh

```diff
@@ -29,9 +29,6 @@
 source $(dirname $0)/e2e-common.sh
 
 function knative_setup() {
-  if is_ingress_class istio; then
-    install_istio || fail_test "Istio installation failed"
-  fi
```

Comment:
Is it intentional to remove the installation of Istio completely?

Reply:
right, it has been run twice.